.github/workflows/workflow.yml

```diff
@@ -60,7 +60,7 @@ jobs:
           key: build-${{ env.pythonLocation }}-${{ hashFiles('setup.py') }}-${{ github.sha }}
 
       - name: Run openfisca-core tests
-        run:  make test-core
+        run:  make test-core openfisca_args="--workers auto"
```

Comment:
Does this make sense when there is always only one core in CI?

Reply:
I think there are two actually. Haven't checked though.